src/dispatchers/mouseDispatcher.ts

```diff
@@ -8,11 +8,12 @@ export module Dispatchers {
     private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Mouse";
     private translator: Utils.ClientToSVGTranslator;
     private _lastMousePosition: Point;
-    private _moveBroadcaster: Core.Broadcaster<Dispatchers.Mouse>;
-    private _downBroadcaster: Core.Broadcaster<Dispatchers.Mouse>;
-    private _upBroadcaster: Core.Broadcaster<Dispatchers.Mouse>;
-    private _wheelBroadcaster: Core.Broadcaster<Dispatchers.Mouse>;
-    private _dblClickBroadcaster: Core.Broadcaster<Dispatchers.Mouse>;
+
+    private _moveCallbackSet: Utils.CallbackSet<Function>;
+    private _downCallbackSet: Utils.CallbackSet<Function>;
+    private _upCallbackSet: Utils.CallbackSet<Function>;
+    private _wheelCallbackSet: Utils.CallbackSet<Function>;
+    private _dblClickCallbackSet: Utils.CallbackSet<Function>;
```

Comment:
We can get more specific here and probably use `<MouseCallback>` for the generic.